nrf_wifi/os_if/inc/osal_structs.h

```diff
@@ -12,7 +12,16 @@
 #ifndef __OSAL_STRUCTS_H__
 #define __OSAL_STRUCTS_H__
 
+#ifdef __ZEPHYR__
 #include <stddef.h>
+#include <string.h>
+#include <stdbool.h>
+#include <stdarg.h>
+#elif __KERNEL__
```

Comment:
Add a comment that this is for Linux kernel.

Reply:
Yeah, updated